app/assets/stylesheets/ltags/GithubTag.scss

```diff
@@ -337,22 +335,22 @@
 .gh-btn-container {
   text-align: center;
   padding: 0.35em 0 1.35em;
-  box-shadow: -0px -0px 60px 42px #fff;
-  box-shadow: -0px -0px 60px 42px var(--card-bg);
+  box-shadow: -0 -0 60px 42px var(--card-bg, #fff);
```

Comment:
Any reason for providing fallback value for this var? All themes come with these values and they are defined in [default colors file](https://github.com/forem/forem/blob/master/app/assets/stylesheets/config/_colors.scss) or appropriate one for [specific theme](https://github.com/forem/forem/tree/master/app/assets/stylesheets/themes). So there should never be a case when this variable would need a fallback, I think...

Reply:
Fixed